pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -0,0 +1,86 @@
+import warnings
+from functools import singledispatch
+
+import torch
+
+from pytensor.compile.ops import DeepCopyOp
+from pytensor.graph.fg import FunctionGraph
+from pytensor.ifelse import IfElse
+from pytensor.link.utils import fgraph_to_python
+from pytensor.raise_op import CheckAndRaise
+
+
+@singledispatch
+def pytorch_typify(data, dtype=None, **kwargs):
+    r"""Convert instances of PyTensor `Type`\s to PyTorch types."""
+    if dtype is None:
+        return torch.tensor(data)
```

Comment:
What about all `torch.as_tensor`? Does it accept `dtype=None`?

Reply:
Yes, it accepts `dtype=None`

Reply:
Let's just use `as_tensor` then